index.js

```diff
@@ -106,6 +106,43 @@ var Controllers = {
     };
   },
 
+  TabBarControllerIOS: function (id) {
+    return {
+      setHidden: function (params) {
+        return RCCManager.TabBarControllerIOS(id, "setTabBarHidden", params);
+      }
+    };
+  },
+
+  Modal: {
+    showLightBox: function(componentId, style) {
+      if (style.backgroundColor !== undefined) {
+        style.backgroundColor = processColor(style.backgroundColor);
+      }
+      RCCManager.modalShowLightBox(componentId, style);
+    },
+    dismissLightBox: function() {
```

Comment:
Same comment as before:
`dismissLightBox: function(params)`
And call it like:
```js
Modal.dismissLightBox({
  animated: true
});
```
or just `Modal.dismissLightBox();` - very similar to how `Controllers.NavigationControllerIOS(id).pop()` works